dockstore-webservice/src/main/java/io/dockstore/webservice/permissions/InMemoryPermissionsImpl.java

```diff
@@ -61,7 +60,7 @@
     private DockstoreWebserviceConfiguration configuration;
 
     @Override
-    public List<Permission> setPermission(Workflow workflow, User requester, Permission permission) {
+    public List<Permission> setPermission(User requester, Workflow workflow, Permission permission) {
```

Comment:
any particular reason for this swap?

Reply:
Should have mentioned in the commit message... I did this to make the method signature similar to other methods in the interface, where the User is the first param.